Clocks/FMI3.xml

```diff
@@ -1,42 +1,39 @@
  <ScheduledExecution modelIdentifier="Clocks"/>
-  <LogCategories>
-   <Category name="logEvents" description="Log events"/>
-   <Category name="logStatusError" description="Log error messages"/>
-  </LogCategories>
-  <DefaultExperiment stopTime="10" stepSize="1"/>
-   <ModelVariables>
-    <Clock name="inClock1" valueReference="1001" causality="input" interval="constant" priority="0" start="0"/>
-    <Clock name="inClock2" valueReference="1002" causality="input" interval="fixed" priority="1" start="0"/>
-    <Clock name="inClock3" valueReference="1003" causality="input" interval="countdown" priority="2" start="0"/>
-    <Clock name="outClock2" valueReference="1005" causality="output" interval="triggered" priority="2"/>
-
-    <Int32 name="inClock1Ticks" valueReference="2001" causality="output"/>
-    <Int32 name="inClock2Ticks" valueReference="2002" causality="output"/>
-    <Int32 name="inClock3Ticks" valueReference="2003" causality="output"/>
-    <Int32 name="totalInClockTicks" valueReference="2004" causality="output"/>
-    <Int32 name="result2" valueReference="2005" causality="output"/>
-    <Int32 name="input2" valueReference="2006" causality="input" start="0"/>
-    <Int32 name="output3" valueReference="2007" causality="output"/>
-   </ModelVariables>
-   <ModelStructure>
-    <Output valueReference="1004"/>
-    <Output valueReference="1005"/>
-    <Output valueReference="2001"/>
-    <Output valueReference="2002"/>
-    <Output valueReference="2003"/>
-    <Output valueReference="2004"/>
-    <Output valueReference="2005"/>
-    <Output valueReference="2007"/>
-    <InitialUnknown valueReference="1004"/>
-   </ModelStructure>
+ <LogCategories>
+  <Category name="logEvents" description="Log events"/>
+  <Category name="logStatusError" description="Log error messages"/>
+ </LogCategories>
+ <DefaultExperiment stopTime="10" stepSize="1"/>
+ <ModelVariables>
+  <Clock name="inClock1" valueReference="1001" causality="input" interval="constant" intervalDecimal="1.0" priority="0"/>
+  <Clock name="inClock2" valueReference="1002" causality="input" interval="triggered" priority="1"/>
+  <Clock name="inClock3" valueReference="1003" causality="input" interval="countdown" priority="2"/>
+  <Clock name="outClock" valueReference="1005" causality="output" interval="triggered" priority="2"/>
+  <Int32 name="inClock1Ticks" valueReference="2001" causality="output"/>
+  <Int32 name="inClock2Ticks" valueReference="2002" causality="output"/>
+  <Int32 name="inClock3Ticks" valueReference="2003" causality="output"/>
+  <Int32 name="totalInClockTicks" valueReference="2004" causality="output"/>
+  <Int32 name="result2" valueReference="2005" causality="output"/>
+  <Int32 name="input2" valueReference="2006" causality="input" start="0"/>
+  <Int32 name="output3" valueReference="2007" causality="output"/>
+ </ModelVariables>
+ <ModelStructure>
+  <Output valueReference="1005"/>
+  <Output valueReference="2001" dependencies="1001"/>
```

Comment:
Do we really list clocks in the list of dependencies of outputs?
We have to specify these dependenencies in the "clock" element in the model struture, which is missing currently

Reply:
Thanks for checking so carefully. I will adjust this ...

Reply:
Thanks for fixing. 
Just saw, that in https://fmi-standard.org/docs/3.0-dev/#ModelStructure it is clearly defined that output clocks should be listed as outtputs as you did